.ci/static-checks.sh

```diff
@@ -309,7 +309,7 @@ static_check_rust_arch_specific()
 		return
 	fi
 
-	find . -type f -name "*.rs"  | egrep -v "target/|grpc-rs/|protocols/" | xargs rustfmt --check > /dev/null 2>&1
+	{ find . -type f -name "*.rs"  | egrep -v "target/|grpc-rs/|protocols/" | xargs rustfmt --check > /dev/null 2>&1; }
```

Comment:
I think you are missing a `|| true` at the end of the `{ ... }`? Currently, if this line fails, the script will still exit. Is that what you want?

Reply:
I think @jodh-intel is right - you probably want to copy the code from around line 453, which looks like the below. Note both the capture of `$?` into a variable, and the use of `|| true` to avoid the unexpected `set -e` quit...

```bash
        local ret

	{ curl -sIL --max-time "$url_check_timeout_secs" --retry "$url_check_max_tries" "$url" &>"$curl_out"; ret=$?; } || true

	# A transitory error, or the URL is incorrect,
	# but capture either way.
	if [ "$ret" -ne 0 ]; then
		echo "$url" >> "${invalid_file}"

		die "check failed for URL $url after $url_check_max_tries tries"
	fi
```

Reply:
@jodh-intel  @grahamwhaley  updated!